File: msgraph-mail/dotnet/Users/BodyType.cs
using Kiota.Abstractions.Serialization;
using System;
using System.Collections.Generic;
using System.Linq;
namespace Graphdotnetv4.Users {
    public class BodyType : IParsable<BodyType> {
        public IDictionary<string, Action<BodyType, IParseNode>> DeserializeFields => new Dictionary<string, Action<BodyType, IParseNode>> {
        };
        public void Serialize(ISerializationWriter writer) {
        }
    }
}

File: msgraph-mail/typescript/src/users/bodyType.ts
import {SerializationWriter, ParseNode, Parsable} from '@microsoft/kiota-abstractions';

export class BodyType implements Parsable<BodyType> {
    public serialize (writer: SerializationWriter) : void {
    };
    public deserializeFields () : Map<string, (item: BodyType, node: ParseNode) => void> {
        return new Map<string, (item: BodyType, node: ParseNode) => void>([
        ]);
    };
}
